test/e2e/container_probe.go

```diff
@@ -43,7 +43,7 @@ var _ = Describe("Probing container", func() {
 		expectNoError(err)
 		startTime := time.Now()
 
-		Expect(wait.Poll(poll, 90*time.Second, func() (bool, error) {
+		Expect(wait.Poll(poll, 120*time.Second, func() (bool, error) {
```

Comment:
This has to be > 30s since the initial delay is 30s. How about we settle for 60s?